ui/apps/platform/src/Containers/Integrations/hooks/useIntegrationActions.ts

```diff
@@ -52,6 +53,9 @@ function useIntegrationActions(): UseIntegrationActionsResult {
                 responseData = await generateAPIToken(data);
             } else if (type === 'clusterInitBundle') {
                 responseData = await generateClusterInitBundle(data);
+            } else if (type === 'machineAccess') {
+                responseData = await createMachineAccessConfig(data);
+                history.goBack();
```

Comment:
Other two types are cluster init bundles and API tokens. 

Cluster init bundles: IIRC, after creation the generated certificate files are displayed only once (since we don't save them to the database). If we would go back, customer wouldn't be able to read and use certificates, rendering generated init bundle useless.

API tokens: same, but with the token value